impl/client/src/main/javascript/web/pentaho/visual/samples/calc/model.js

```diff
@@ -57,9 +57,9 @@ define([
           {
             name: "operation",
             type: {
-              base: "refinement",
-              of:   "string",
-              facets: ["DiscreteDomain"],
+              base: "string",
```

Comment:
In the other models you omitted `base: "string"` because it is the default (right?).
You kept it here because it is the sample?